DailyReportWeb/Controllers/Api/PolicyController.cs

```diff
@@ -40,6 +56,18 @@ public void Put(string id, [FromBody]Policy updatedPolicy)
             // TODO: save the updated policy
         }
 
+		public void Put([FromBody]PolicySummary policySummary)
+		{
+			var policy = new Policy();
+			policy.BaseUrl = policySummary.baseUrl;
+			policy.SharedSecret = policySummary.sharedSecret;
+			policy.ProjectId = Int32.Parse(policySummary.id);
+			//policy.ProjectKey = policySummary.key;
+			policy.ReportTime = policySummary.time;
+
+			policy.SaveToFile(@"C:\Workspace\DailyReportTool\JiraReporter\Policies\testing.txt");
```

Comment:
calea e setata precum work-in-progress doar pentru a verifica functionalitatea, in momentul in care se decid detaliile precum numele, putem alege:
key="policyPath" value="C:\Apps\etc"